src/main/java/com/breskul/hw/controllers/NasaPictureController.java

```diff
@@ -0,0 +1,57 @@
+package com.breskul.hw.controllers;
+
+import com.breskul.hw.endpoints.dto.LargestNasaPictureRequest;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+@Component
+public class NasaPictureController {
+
+    Logger logger = LoggerFactory.getLogger(NasaPictureController.class);
+
+    @Autowired
+    RabbitTemplate rabbitTemplate;
+
+    @Autowired
+    NasaController nasaController;
+    Map<String, byte[]> biggestPictures = new HashMap<>();
+
+    public String requestTheBiggestPicture(int sol, String camera) {
+        String commandId = RandomStringUtils.random(10, true, true);
+
+        LargestNasaPictureRequest request = new LargestNasaPictureRequest();
+        request.setSol(sol);
+        request.setCamera(camera);
+        request.setCommandId(commandId);
+
+        rabbitTemplate.convertAndSend("picture-exchange", "", request);
+
+        logger.info("Command send to Rabbit. CommandId: " + commandId);
```

Comment:
You can use the built-in string formatter according to the `info` method signature, instead of string concatenation -> ```logger.info("Command sent to Rabbit. CommandId: {} ", commandId);```